src/python/pants/backend/jvm/ivy_utils.py

```diff
@@ -17,6 +16,7 @@
 from functools import total_ordering
 
 import six
+from pex.common import safe_copy
```

Comment:
I'm all for code re-use, but this seems off. For one, jvm code depending on pex doesn't make sense. Probably more important though, I'm pretty sure pex.common is not intended to be public.

Reply:
Done